pkg/vulnsrc/osv/osv.go

```diff
@@ -123,6 +123,11 @@ func (vs VulnSrc) save(eco ecosystem, entries []Entry) error {
 }
 
 func (vs VulnSrc) commit(tx *bolt.Tx, eco ecosystem, entry Entry) error {
+
+	if entry.Withdrawn != nil {
```

Comment:
maybe len() will be better?
```suggestion
	if len(entry.Withdrawn) > 0 {
```

Reply:
`Withdrawn` is a pointer to time.Time

Reply:
oh, ok.

Reply:
Do we want to compare the date? The following code doesn't work probably, but I just wanted to explain my thought.

```suggestion
	if entry.Withdrawn != nil && entry.Withdrawn.Before(time.Now()){
```